src/encode.cpp

```diff
@@ -1,6 +1,9 @@
 #include "encode.h"
 #include "abstractops.h"
+#include "mlir/IR/Types.h"
+#include "mlir/Support/LLVM.h"
```

Comment:
Let's move these includes to line 7~ below.

Reply:
Fixed